tsconfig.json

```diff
@@ -1,9 +1,19 @@
 {
   "compilerOptions": {
+    "jsx": "react",
     "strict": true,
-    "target": "es5",
-    "lib": ["es5", "dom"],
-    "types": ["node", "jest"]
+    "esModuleInterop": true,
+    "target": "ES2019", //Until webpack can handle JS optional chaining syntax
+    "lib": ["esnext", "dom"],
+    "types": ["node", "jest", "@testing-library/jest-dom"],
+    "allowJs": true,
+    "resolveJsonModule": true,
+    "moduleResolution": "node",
+    "baseUrl": "./",
+    "paths": {
+      "@/Styling": ["client/Bootstrap/Styling.scss"],
```

Comment:
As per the Webpack approach, Styling was a relative alias - ie it resolved to a `Styling.scss` in that directory - not a top level file. I still have this as a relative alias from webpack in my server work, and have no path entry for it in tsconfig

Reply:
It'll need to be in tsconfig for jest purposes